### ISSUE TITLE
MSS.__init__ throws AttributeError due to SetProcessDPIAware on XP

### ISSUE THREAD

**Issue by foone** [BoboTiG/python-mss#109]
General information:

* OS name: _Windows_
* OS version: __XP 5.1.2600__
* OS architecture: _32 bits_
* Resolutions:
    * Monitor 1: _1920x975_
* Python version: _2.7.13_
* MSS version: __4.0.1__

### Description of the warning/error

MSS.\_\_init\_\_ fails to initialize on XP because when SetProcessDpiAwareness fails, it falls back to SetProcessDPIAware, which was added in Vista. Wrapping it in another try/except for the Attribute error makes it work fine. (I tried to make a pull request but ran into issues with github's UI and gave up, but it's a minor fix)

### Full message

```
Traceback (most recent call last):
  File "sc2k.py", line 58, in <module>
    take_screenshot(filename)
  File "sc2k.py", line 30, in take_screenshot
    with mss.mss() as sct:
  File "C:\Python27\lib\site-packages\mss\factory.py", line 38, in mss
    return MSS(**kwargs)
  File "C:\Python27\lib\site-packages\mss\windows.py", line 92, in __init__
    self.user32.SetProcessDPIAware()
  File "C:\Python27\lib\ctypes\__init__.py", line 379, in __getattr__
    func = self.__getitem__(name)
  File "C:\Python27\lib\ctypes\__init__.py", line 384, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))
AttributeError: function 'SetProcessDPIAware' not found
```

**Comment by BoboTiG** [comment]
Hello,

XP! :)
Would you mind open a PR? It would be great.

**Comment by BoboTiG** [comment]
I just published 4.0.2 with you patch ;)
